reddit_parser.py

```diff
@@ -237,14 +304,14 @@ def NN_encode(self, text):
         #   (4) Map tokens to their IDs.
         id = 0
         for index, sent in enumerate(text):  # iterate over the sentences
-             encoded_sent = self.bert_tokenizer.encode(sent,  # Sentence to encode
```

Comment:
I think the bert stuff should be removed for now

Reply:
Removing

Reply:
The bert functions are still there. Am I misreading something?

Reply:
The bert parsing has been removed, as shown by the red highlight here. Not sure if that's what you meant. However, there are still references to the bert_prep file name -- I will remove those too. If there is something else I missed, please be specific with what you mean by bert functions

Reply:
I'm looking through reddit_parser and there are many many references to the bert_prep fn. I am skeptical of changing all those now without accidentally breaking something else. We made a task for this earlier so I vote we come back to it after this is merged, unless you are absolutely partial to removing those references now

Reply:
> The bert functions are still there. Am I misreading something?

I am noticing this functionality still appears to be there when looking at the diff/comment, although it is removed on my end.